src/kvp/annotation/mod.rs

```diff
@@ -150,6 +150,19 @@ impl TryFrom<BTreeMap<String, String>> for Annotations {
     }
 }
 
+impl<const N: usize, T, K> TryFrom<[(T, K); N]> for Annotations
+where
+    T: AsRef<str>,
+    K: AsRef<str>,
+{
+    type Error = AnnotationError;
+
+    fn try_from(value: [(T, K); N]) -> Result<Self, Self::Error> {
+        let kvps = KeyValuePairs::try_from(value)?;
+        Ok(Self(kvps))
+    }
+}
+
```

Comment:
praise: yes! this will be nice so I don't have to convert BTreeMaps into Labels. :raised_hands:

Reply:
Yeah that's a very nice improvement indeed.